firebase-firestore/src/androidTest/java/com/google/firebase/firestore/ValidationTest.java

```diff
@@ -461,35 +461,42 @@ public void queriesCannotBeSortedByAnUncommittedServerTimestamp() {
     ListenerRegistration listenerRegistration =
         collection.addSnapshotListener(
             (snapshot, error) -> {
-              assertNotNull(snapshot);
```

Comment:
Instead of failing, this test would time out because exceptions within the callback lambda wouldn't be raised on test thread.